include/npcomp/Dialect/Torch/IR/GeneratedAtenOps.td

```diff
@@ -1295,6 +1295,38 @@ def Torch_AtenArangeStartOp : Torch_Op<"aten.arange.start", [
   let assemblyFormat = "$start `,` $end `,` $dtype `,` $layout `,` $device `,` $pin_memory attr-dict `:` type($start) `,` type($end) `,` type($dtype) `,` type($layout) `,` type($device) `,` type($pin_memory) `->` type($result)";
 }
 
+def Torch_AtenArgmaxOp : Torch_Op<"aten.argmax", [
+    AllowsTypeRefinement,
+    HasValueSemantics
+  ]> {
+  let summary = "Generated op for `aten::argmax : (Tensor, int?, bool) -> (Tensor)`";
+  let arguments = (ins
+    AnyTorchTensorType:$self,
+    TorchOptionalIntType:$dim,
+    Torch_BoolType:$keepdim
+  );
+  let results = (outs
+    AnyTorchTensorType:$result
+  );
+  let assemblyFormat = "$self `,` $dim `,` $keepdim attr-dict `:` type($self) `,` type($dim) `,` type($keepdim) `->` type($result)";
+}
+
+def Torch_AtenArgmaxOutOp : Torch_Op<"aten.argmax.out", [
```

Comment:
The .out variant won't work with the code we currently have. ReduceOpVariants needs to learn to support it before it is meaningful to add (it should be handled similar to `trailing_` variants, see emit_with_mutating_variants in torch_ods_gen.py and ReduceOpVariants pass)

Reply:
thanks